test/integration/targets/lookup_varnames/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+# Example copied from docs
+- name: Set some variables
+  set_fact:
+    qz_1: hello
+    qz_2: world
+    qa_1: "I won't show"
+    qz_: "I won't show either"
+
+- name: Try various regexes and make sure they work
+  assert:
+    that:
+      - lookup('varnames', '^qz_.+', wantlist=True) == ['qz_1', 'qz_2']
+      - lookup('varnames', '^qz_.+', '^qa.*', wantlist=True) == ['qz_1', 'qz_2', 'qa_1']
+      - >-
```

Comment:
Is the `>-` just being used to avoid quoting the entire line with `"`?

Reply:
Yeah. Is that bad practice? It makes it slightly easier to read imho.

Reply:
styles are subjective, for me its easier quoted but i don't use syntax colors, which is probably why you prefer this style

Reply:
This is the first time I recall seeing `>-` used in our integration tests to avoid quotes. I don't have strong feelings about it, but I do tend to favor matching existing project style.

Reply:
I guess I'd prefer quoted as well, unless it's being used to avoid escaped quotes like `\"`.